.github/workflows/reusable_qemu.yml

```diff
@@ -1,14 +1,29 @@
 # Builds project on qemu with custom hmat settings
 name: Qemu
 
-on: workflow_call
+on:
+  workflow_call:
+    inputs:
+      short_run:
+        description: Should the workflow run only basic tests?
+        type: boolean
+        default: false
+      os:
+        description: List of OSes
+        type: string
+        default: '["ubuntu-23.04"]'
```

Comment:
Why are we testing non-LTS OS here?

Reply:
good question, I didn't change that, we used this before and I think as long as it works I would keep it; we can also extend/change the list of OSes if needed (in another PR)

Reply:
That should be dropped in some other PR, I think.